### DISCUSSION TITLE
No default navigation

### DISCUSSION THREAD

**Discussion by happylolonly** [wojtekmaj/react-pdf#1401] [Q&A]
Why such lib doesn't have default navigation out of box?
Or if render all pages, scroll also not working ok.

I planned quickly add it, without such things, or I missed something?

**Comment by wojtekmaj** [comment]
Because it's not the point of this package to provide Adobe Reader alternative, but an unopinionated way to build PDF functionalities in your React app.

**Reply by happylolonly** [reply]
Can I render all pages and add scroll somehow? Now not working ok

**Reply by wojtekmaj** [reply]
Of course. Treat Page like img. Just render an array of pages, style as you like.
